database/install.sh

```diff
@@ -18,6 +10,8 @@ sed -i "s|DB_HOST|$DB_HOST|g" infrastructure/database/create.sql
 sed -i "s|DB_USER|$DB_USER|g" infrastructure/database/delete.sql
 sed -i "s|DB_PASS|$DB_PASS|g" infrastructure/database/delete.sql
 sed -i "s|DB_HOST|$DB_HOST|g" infrastructure/database/delete.sql
+
+service mysql start
```

Comment:
was this needed for the mysql command below to run?

Reply:
Yeah, at least I got an error without it running

Reply:
Can do some more investigation